lib/rules/key-spacing.js

```diff
@@ -72,6 +72,27 @@ function isSingleLine(node) {
     return (node.loc.end.line === node.loc.start.line);
 }
 
+/** Sets option values from the configured options with defaults
+ * @param {Object} toOptions Object to be initialized
+ * @param {Object} fromOptions Object to be initialized from
+ * @param {Object} defultOptions Object to be used for default values
+ * @returns {Object} The object with correctly initialized options and values
+ */
+function initOptions(toOptions, fromOptions, defultOptions) {
+    toOptions.mode = fromOptions.mode || defultOptions.mode || "strict";
+    if (typeof fromOptions.beforeColon !== "undefined") {
+        toOptions.beforeColon = +!!fromOptions.beforeColon;
```

Comment:
I'm slightly confused as to what does +boolean supposed to do. Can you explain please?